.github/workflows/scripts/win-test-services.ps1

```diff
@@ -44,7 +44,9 @@ $expected_svc_env_vars = @{
   "SPLUNK_REALM"            = "$realm";
   "SPLUNK_API_URL"          = "$api_url";
   "SPLUNK_INGEST_URL"       = "$ingest_url";
-  "SPLUNK_TRACE_URL"        = "${ingest_url}/v2/trace";
+  # We no longer test for this as we have moved to OTLP and it breaks upgrade tests.
+  # We have a configuration converter in place to catch old values as part of the collector startup.
+  #"SPLUNK_TRACE_URL"        = "${ingest_url}:443;
```

Comment:
Hum, this can be changed in the test, as long as we track the first version doing the switch. Anyway, much better as a follow-up then in this PR.

Reply:
Note 443 is not optional right now.